installer/updater/updater.cc

```diff
@@ -301,11 +304,13 @@ struct Updater {
   void set_progress(std::string text) {
     std::lock_guard<std::mutex> guard(lock);
     progress_text = text;
+    printf("%s\n", text.c_str());
   }
 
   void set_error(std::string text) {
     std::lock_guard<std::mutex> guard(lock);
     error_text = text;
+    printf("%s\n", text.c_str());
```

Comment:
`std::cout`?

Reply:
When in Rome. There's a lot of other C-isms in updater.cc if we're going to go there.